### PR TITLE
#184347014 Setup swagger documentation

### PR DESCRIPTION
**Pull request by Aline-Uwera**
#### What does this PR do?
Setup API documentation using swagger
#### Description of Task to be completed?
- [x] configure swagger documentation
- [x] add pull request template
#### How should this be manually tested?
- Checkout to this branch [chore/setup-swagger-docs-184347014](https://github.com/atlp-rwanda/vikings-ec-bn/tree/chore/setup-swagger-docs-184347014)
- Run `npm install`
- Run `npm run dev`
- Open browser and visit http://localhost:5000/api-docs/#
#### Any background context you want to provide?
*The API documentation is configured to be placed under documentation folder which is found in src folder and this documentation is written in javascript files which makes it easier to deal with and each resource being documented should have it own file to keep the code clean. after you have created your documentation file make sure you add it into index.js file in documentation folder to let the changes you have done take place.*
#### What are the relevant pivotal tracker stories?
[#184347014](https://www.pivotaltracker.com/story/show/184347014)
#### Screenshots (if appropriate)
Screenshot from 2023-02-02 00-20-26
#### Questions:
N/A

### CODE REVIEW

**Review by irakozetony — APPROVED** [review]
I think that .github folder is not supposed to be here!

**Review by Abdden — APPROVED** [review]

**Review by nshimiyejayd — APPROVED** [review]

**Review by Abdden — APPROVED** [review]

File: src/routes/router.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const router = express.Router();
+
+
+
+router.get('/', (req, res) =>{
+    res.send("Welcome!")
+});
+
+export default router;
```
Comment (ncutixavier) [inline src/routes/router.js:11]: In the routes folder, let's rename the file by `*.route.js`. For example, If you are implementing authentication routes, you can name it `authentication.route.js`

File: tests/test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+beforeEach(async() => {
+    await connectDB();;
+  });
+let team="Vikings"
+
+ test("should respond the team name",()=>{
+            
+            expect(team).toBe("Vikings")
+  })
```
Comment (ncutixavier) [inline tests/test.js:11]: Same here, let's follow this convention `*.test.js`

**Review by ncutixavier — CHANGES_REQUESTED** [review]
Remember to squash your commits

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**